src/firefox/remote.js

```diff
@@ -151,3 +196,10 @@ export class RemoteFirefox {
       });
   }
 }
+
+// NOTE: this fix an issue with flow and default exports (which currently
+// lose their type signatures) by explicitly declare the default export
+// signature. Reference: https://github.com/facebook/flow/issues/449
+declare function exports( // eslint-disable-line no-unused-vars
+  port: number, extraParams?: ConnectExtraParams
+): Promise<RemoteFirefox>;
```

Comment:
In the link the workaround seems to be applied into one of those external declaration files, I'm trying to avoid to put the type declarations in a separate file that is magically included in the flow checks, and it seems that we can put declarations for a module in the module itself instead by just skipping the "declare module" container.

I agree, I'm going to move all the type declarations near to where they are used.